osbs/core.py

```diff
@@ -872,6 +872,23 @@ def restore_resource(self, resource_type, resource):
         check_response(response)
         return response
 
+    def create_config_map(self, config_data):
+        url = self._build_k8s_url("configmaps/")
+        response = self._post(url, data=json.dumps(config_data))
+        check_response(response)
+        return response
+
+    def get_config_map(self, config_name):
+        url = self._build_k8s_url("configmaps/")
+        response = self._get(url, data=json.dumps(config_name))
+        check_response(response)
+        return response
+
+    def delete_config_map(self, config_name):
+        url = self._build_k8s_url("configmaps/")
+        response = self._delete(url, data=json.dumps(config_name))
```

Comment:
DeleteOptions has 5 parameters, all optional. Should the caller to delete_config_map pass them in or should I set them in delete_config_map? in which case I'm just passing 'body = {}' as far as I can tell.